src/web_analysis/data/prompt_templates.json

```diff
@@ -0,0 +1,44 @@
+    },
+    {
+      "id": "scraping-policy-keywords",
+      "content": "Question: Does the website restrict scraping? Carefully review the following text from the Terms of Service (TOS) document and determine if the statement is True: {}. If the statement is True, provide the exact text that answers the question as evidence. If the statement is False, set evidence as \"N/A\". Return only a dictionary with your verdict and the corresponding evidence. Format the response like this to ensure it can be loaded by JSON: {{\"verdict\": \"True\", \"evidence\": \"Exact text from TOS.\"}} or {{\"verdict\": \"False\", \"evidence\": \"N/A\"}}"
+    },
+    {
+      "id": "AI-policy",
+      "content": "Question: Does website have specific restrictions related to AI training? Carefully review the following Terms of Service (TOS) document and determine if the statement is True: {}. If the statement is True, provide the exact text from the TOS that answers the question as evidence. If the statement is False, set evidence as \"N/A\". Return only a dictionary with your verdict and the corresponding evidence. Format the response like this to ensure it can be loaded by JSON: {{\"verdict\": \"True\", \"evidence\": \"Exact text from TOS.\"}} or {{\"verdict\": \"False\", \"evidence\": \"N/A\"}}"
+    },
+    {
+      "id": "AI-policy-keywords",
+      "content": "Question: Does website have specific restrictions related to AI training? Carefully review the following text from the Terms of Service (TOS) document and determine if the statement is True: {}. If the statement is True, provide the exact text that answers the question as evidence. If the statement is False, set evidence as \"N/A\". Return only a dictionary with your verdict and the corresponding evidence. Format the response like this to ensure it can be loaded by JSON: {{\"verdict\": \"True\", \"evidence\": \"Exact text from TOS.\"}} or {{\"verdict\": \"False\", \"evidence\": \"N/A\"}}"
+    },
+    {
+      "id": "competing-services",
+      "content": "Question: Does website restrict use of content for competing services? Carefully review the following Terms of Service (TOS) document and determine if the statement is True: {}. If the statement is True, provide the exact text from the TOS that answers the question as evidence. If the statement is False, set evidence as \"N/A\". Return only a dictionary with your verdict and the corresponding evidence. Format the response like this to ensure it can be loaded by JSON: {{\"verdict\": \"True\", \"evidence\": \"Exact text from TOS.\"}} or {{\"verdict\": \"False\", \"evidence\": \"N/A\"}}"
```

Comment:
something like this maybe for more specificity: "use of content in any way, such as for commercial or competing services, or only for academic use"